.github/workflows/update_version.yml

```diff
@@ -76,37 +58,16 @@ jobs:
 
       # update all the version strings
       - name: Update version
-        if: github.event_name == 'workflow_dispatch'
         run: python scripts/update_version.py
 
-      # create a pull request updating versions in develop
-      - name: Create Pull Request
-        id: version_pr
-        uses: peter-evans/create-pull-request@v6
-        with:
-          delete-branch: true
-          branch-suffix: short-commit-hash
-          base: develop
-          commit-message: Update version to ${{ env.VERSION }}
-          title: Bump to ${{ env.VERSION }}
-          body: |
-            - [x] Update to ${{ env.VERSION }}
-            - [ ] Check the [release workflow](https://github.com/pybamm-team/PyBaMM/blob/develop/.github/release_workflow.md)
-
-      # checkout to the version branch for the final release
-      - uses: actions/checkout@v4
-        if: github.event_name == 'workflow_dispatch' && !startsWith(github.event.inputs.append_to_tag, 'rc')
-        with:
-          ref: '${{ env.NON_RC_VERSION }}'
```

Comment:
This would still be required to update the version in develop, right?

Reply:
I was trying to go for something like this: 
![image](https://github.com/user-attachments/assets/af259399-197f-4877-a0b8-0301e5049aa3)

Where if the releases were fast, then the release branch goes right to main, then main updates develop for the changelog/version.

Then if we do a bugfix release, it is manual anyway